### PR TITLE
Eric lasers

### PR DESCRIPTION
**Pull request by yangerdanger**
Implemented laser objects into the tileset as well as a lethal tag for tiled, and deleted some levels that no longer work.

### CODE REVIEW

File: core/src/edu/cornell/gdiac/rabbeat/ObjectController.java [hunk 1]
```diff
@@ -498,12 +503,21 @@ public void populateObjects(Vector2 scale) {
                             float y = platform.getFloat("y");
                             Vector2 dim = new Vector2(platform.getFloat("width"), platform.getFloat("height"));
                             String align = "";
-                            for (JsonValue prop : platform.get("properties")) {
```
Comment (JacobSeto) [inline core/src/edu/cornell/gdiac/rabbeat/ObjectController.java:501]: You deleted this line of code that is crucial for beatList to work

Comment (yangerdanger) [reply core/src/edu/cornell/gdiac/rabbeat/ObjectController.java:501]: I didn't, I replaced it with a if statement that checked to make sure properties wasn't null first.

Comment (yangerdanger) [reply core/src/edu/cornell/gdiac/rabbeat/ObjectController.java:501]: image

**Review by JacobSeto — APPROVED** [review]
Looks good!

**Pull request merged**